src/runtime-rs/crates/resource/src/cgroups/utils.rs

```diff
@@ -14,3 +25,61 @@
 pub(crate) fn gen_overhead_path(path: &str) -> String {
     format!("kata_overhead/{}", path.trim_start_matches('/'))
 }
+
+pub(crate) fn rename_cgroup_path(path: &str) -> String {
```

Comment:
1. The `rename_cgroup_path()` seems like a generic function name, and the implementation here is to add the prefix kata for cgroup path, suggest rename like `add_kata_prefix_for_cgroup_path()`.
2. Suggest adding a comment for this function.

Reply:
renamed and added a comment